src/coreComponents/common/logger/ErrorHandling.hpp

```diff
@@ -230,6 +235,24 @@ class ErrorLogger
     bool m_isValidStackTrace = false;
   };
 
+  /**
+   * @brief Retrieve all informations from the ErrorLogger and format into an ascii message
+   * @param errMsg Class containing all the error/warning information
+   * @param rank The rank where the error/warning happened
+   * @return The error message displayed to the standard output
+   */
+  static string formatMsgToAscii( ErrorLogger::ErrorMsg const & errMsg, string const & rank )
```

Comment:
Follow the function goal: "formatX", and not "logX" (and stick to "formatX" since that is more flexible).
```suggestion
  static void formatMsgToAscii( std::ostream & outStream, ErrorLogger::ErrorMsg const & errMsg, string const & rank )
```